api/V1/systemError.ts

```diff
@@ -0,0 +1,365 @@
+// Fuzzily checks that error logs are similar
```

Comment:
It feels like you could replace most of this code with a [levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance) implementation.  There are at least a couple of implementations on npm.

Reply:
Not sure that any of them would do it like thist. Want to match per word and get substrings not with substitutions and not character based. I think this works best for logs, as we rarely or never should have words that are spelt differently in 2 logs

Reply:
Sorry, I thought the intention was just to get a fuzzy match metric for two strings.
If the use case is more nuanced than that I'd probably want to see some reasonably comprehensive tests that this code works as intended - since there's quite lot of logic going on here.